examples/js/controls/OrbitControls.js

```diff
@@ -58,6 +58,9 @@
 			this.enableZoom = true;
 			this.zoomSpeed = 1.0; // Set to false to disable rotating
 
+			//Set to enable dollying(zooming) with the follow of the mouse pointer on screen.
+			this.enableFollowZoom = false;
```

Comment:
I think this is usually called something like "zoom to cursor" so this might be better: 

`this.zoomToCursor= false;`

Reply:
okay, I have updated it, but I have reservations.

Reply:
emmm is there any other suggestion?  or should I just close the PR? But I don't think such feature is useless.